build.gradle

```diff
@@ -48,9 +62,6 @@ tasks.withType(JavaCompile).configureEach {
 	// see http://yodaconditions.net/blog/fix-for-java-file-encoding-problems-with-gradle.html
 	// If Javadoc is generated, this must be specified in that task too.
 	it.options.encoding = "UTF-8"
-
-	// Minecraft 1.17 (21w19a) upwards uses Java 16.
-	it.options.release = 16
```

Comment:
Why the removal?

Reply:
you dont need to specify release version since ` ​targetCompatibility ​=​ ​JavaVersion​.​VERSION_16 ` is written.